tests/unit/ct-completion.test.ts

```diff
@@ -0,0 +1,243 @@
+import { describe, expect, test, vi, afterEach } from "vitest";
+import { getCompletionPositionForLine, generateCompletionData } from "./utils";
+
+// Whole coded test
+
+const unitTst = `
```

Comment:
I think this file can likely be empty; it doesn't need to have all of this stuff in it.

If it can be empty, let's use an empty file.

Reply:
Used empty file in 805ebe742f3dfb6d82093c3d9606af7a90511b9f .